core-primitives/stf-executor/src/executor_tests.rs

```diff
@@ -279,7 +279,7 @@ pub fn get_stf_state_works() {
 }
 
 pub fn upon_false_signature_get_stf_state_errs() {
-	let sender = AccountId::default();
+	let sender = AccountId::from([0; 32]);
```

Comment:
So they removed the `Default` implementation for a couple of struct, such as `AccountId`?

Reply:
Yes. They removed it for security purposes, as far as I understand. A default account always comes with some security risks.